composer.json

```diff
@@ -3,7 +3,7 @@
   "description": "OpenTelemetry PHP distribution for Uptrace",
   "type": "library",
   "require": {
-    "open-telemetry/opentelemetry": "0.0.16",
+    "open-telemetry/opentelemetry": "1.0.0beta1",
```

Comment:
This meta-package's dependencies still have their 0.0.16 versions still available. I would suggest including the dependencies explicitly to roll back

Reply:
Done, thanks for the hint!